pdc/apps/compose/lib.py

```diff
@@ -149,25 +153,29 @@ def compose__import_rpms(request, release_id, composeinfo, rpm_manifest):
             for srpm_nevra, rpms in rm.rpms.get(variant.uid, {}).get(arch, {}).iteritems():
                 sources.add(srpm_nevra)
                 for rpm_nevra, rpm_data in rpms.iteritems():
-                    imported_rpms += 1
                     path, filename = os.path.split(rpm_data['path'])
                     rpm_id = get_or_insert_rpm(rpms_in_db, cursor, rpm_nevra, srpm_nevra, filename)
                     sigkey_id = common_models.SigKey.get_cached_id(rpm_data["sigkey"], create=True)
                     path_id = models.Path.get_cached_id(path, create=True)
                     content_category = rpm_data["category"]
                     content_category_id = repository_models.ContentCategory.get_cached_id(content_category)
-                    insert_compose_rpms_if_nonexist(compose_rpms_in_db, cursor,
-                                                    var_arch_obj.id, rpm_id,
-                                                    content_category_id, sigkey_id, path_id)
+                    inserted = insert_compose_rpms_if_nonexist(compose_rpms_in_db, cursor,
+                                                               var_arch_obj.id, rpm_id,
+                                                               content_category_id, sigkey_id, path_id)
+                    if inserted:
+                        imported_rpms += 1
```

Comment:
Minor thing. Maybe it could 
if insert_compose_rpms_if_nonexist(compose_rpms_in_db, cursor,
                                                              var_arch_obj.id, rpm_id,
                                                               content_category_id, sigkey_id, path_id):
     imported_rpms += 1